src/readline.cpp

```diff
@@ -70,13 +70,15 @@ class real_readline : public IReadLine
     {
         rl_readline_name = "sdcv";
         using_history();
-        const std::string histname = std::string(g_get_home_dir()) + G_DIR_SEPARATOR + ".sdcv_history";
+        const gchar *hist_file_str = g_getenv("SDCV_HISTFILE");
```

Comment:
It would be nice move this duplicate code (here and in `~real_readline()`) to separate
function.